src/chart.ts

```diff
@@ -60,12 +79,32 @@ export class Chart extends Construct {
    */
   private readonly _labels?: { [name: string]: string };
 
+  /**
+   * Whether or not this chart includes construct metadata.
+   */
+  public readonly includesConstructMetadata: boolean;
```

Comment:
A more appropriate name for this would be simply `constructMetadata`, but I wanted to reserve that in case we ever wanted to use it for an actual object that allows consumers to add their own metadata.

Reply:
Removed this